docs/UserGuide.adoc

```diff
@@ -55,35 +55,36 @@ e.g. typing **`help`** and pressing kbd:[Enter] will open the help window.
   - **Help**: `help`
   - **Exit**: `exit`
   - **Navigate**: `pp` / `np`
-  - **Display**: `display SYSTEM`
+  - **Display**: `display SYSTEM` +
+  e.g. `display p` +
+  e.g. `display s` +
+  e.g. `display i`
 . Pet Tracker System
   - **Add pet**: `addpet n/NAME a/AGE s/SPECIES b/BREED g/GENDER f/FOOD [t/TAG]...` +
   e.g. `addpet n/Garfield a/5 s/cat b/ginger tabby g/M t/lazy f/30 t/good for nothing t/lasagna`
-  - **Find pet**: `findpet [n/NAME]... [t/TAG]...` +
-  e.g. `findpet n/garfield n/odie t/jon`
-  - **Tag pet**: `tagpet INDEX t/TAG [t/TAG]...` +
+  - **Find pet**: `findpets [n/NAME]... [t/TAG]...` +
+  e.g. `findpets n/garfield n/odie t/jon`
+  - **Tag pet**: `tagpet i/INDEX t/TAG [t/TAG]...` +
   e.g. `tagpet 2 t/smelly t/noisy t/cuddly`
-  - **Delete pet**: `deletepet INDEX` +
+  - **Delete pet**: `deletepet i/INDEX` +
   e.g. `deletepet 2`
 . Schedule System
-  - **Add slot**: `addslot [i/INDEX] [n/PETNAME] t/TIME d/DURATION` +
-  e.g. `addslot i/1 t/14:00 d/2:00` +
-  e.g. `addslot n/Coco t/"20-2-2020 13:00" d/1:30`
-  - **Delete slot**: `deleteslot [i/INDEX] [n/PETNAME] t/TIME` +
-  e.g. `deleteslot i/1 t/14:00` +
-  e.g. `deleteslot n/Coco t/"20-2-2020 13:00"`
-  - **Edit slot**: `editslot i/INDEX t/TIME [nt/NEW_TIME]` +
-  e.g. `editslot i/1 t/14:00 nt/8:00` +
-  e.g. `editslot i/1 t/14:00 nt/19:00 nd/1:30` +
-  e.g. `editslot i/12 t/”20-2-2020 13:00” nd/2:00`
-  - **Show schedule**: `showslots [t/DATE]` +
-  e.g. `showslots` +
-  e.g. `showslots t/20-2-2020`
+  - **Add slot**: `addslot n/PETNAME t/DATETIME d/DURATION` +
+  e.g. `addslot n/Coco t/16/11/2020 1300 d/90` +
+  e.g. `addslot n/Brian Griffin t/9/2/2020 0307 d/5`
+  - **Find slot**: `findslots [p/PETNAME] [t/DATE]` +
+  e.g. `findslots p/Brian Griffin` +
+  e.g. `findslots t/20/2/2020` +
+  e.g. `findslots p/Coco t/16/11/2020`
+  - **Edit slot**: `editslot i/INDEX [n/PETNAME] [t/DATETIME] [d/DURATION]` +
+  e.g. `editslot 1 t/16/11/2020 1300 d/120` +
+  e.g. `editslot 3 d/45` +
+  e.g. `editslot 12 n/Brian Griffin`
+  - **Check slot**: `checkslot t/DATETIME` +
+  e.g. `checkslot 17/4/2020 1700`
```

Comment:
also, would using 'list' be more consistent with the rest of the project?

Reply:
for the name of the command, i don't think `list` would be more consistent since our program has no commands called `list` or `listXXX` atm, so there is no "parallel" command for it (e.g. `addpet` and `addslot`)

also if the name of the command is `list`, that might imply that the command modifies the list of slots displayed (similar to `list` in AB3), but `checkslot` does not do that and instead simply sends feedback